docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe.rst

```diff
@@ -0,0 +1,112 @@
+Add the power of Apache Flink® to Apache Kafka® for stateful stream processing
+==============================================================================
+
+Whether it's an event-driven application or a data pipeline, Apache Flink® is an excellent choice due to its ability to perform computations at in-memory speed and at any scale. 
```

Comment:
This reads like marketing blurb. Instead, describe the use case, sending metrics to Kafka, processing them with Flink to do filtering or transformation, and writing the output to a different topic.